### DISCUSSION TITLE
Newly created, unmodified create-next-app project doesn't work

### DISCUSSION THREAD

**Discussion by viggobf** [vercel/next.js#41195] [Help]
Hi, I'm new to Next.js, switching from Gatsby. To start, I simply ran `npx create-react-app@latest`, which ran successfully. Then I did the usual and switched to the new project directory, and ran `next dev`.

Visiting localhost:3000, the basic default-created Next app doesn't even work, and throws the following error:
![image](https://user-images.githubusercontent.com/65341832/194122682-74ede65d-477b-41dc-a96d-4fe175e04223.png)
 What is going on? Am happy to provide further info if needed.

Next: 12.3.1, macOS 12, React/ReactDOM 18.2.0

**Comment by gaston-flores** [comment]
>  start, I simply ran npx create-react-app@latest

Did you mean `create-next-app`?

I am not able to reproduce this in my local environment. Here's my `package.json`:

```json
{
  "name": "my-app",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "next": "12.3.1",
    "react": "18.2.0",
    "react-dom": "18.2.0"
  },
  "devDependencies": {
    "eslint": "8.24.0",
    "eslint-config-next": "12.3.1"
  }
}
```

Can you share a repo that reproduces the issue?

**Reply by viggobf** [reply]
yes, I did mean `create-next-app`. However, I managed to fix the issue by running `npm run dev` instead of `next dev`. Thanks for the message!
